interfaces/Portalicious/eslint.config.js

```diff
@@ -95,6 +95,12 @@ module.exports = tseslint.config(
             'ngSrc',
             'styleClass',
             'position',
+            'pSortableColumn',
+            'field',
+            'display',
+            'sortField',
+            'appendTo',
+            'iconPosition',
```

Comment:
Isn't there a "PrimeNG-ESLint-plugin" somewhere? So we don't have to maintain this list here ourselves?

Reply:
Theoretically we could import a list of attributes from somewhere via code (this is a JS-file with config-as-code after all...

But as that isn't really a commonly used, documented thing (yet), we can make due with this indeed.

---

Some interesting thing I've found:
https://github.com/angular-eslint/angular-eslint/issues/839#issuecomment-1161328555

So we can be more specific to ignore _some_ attributes on _some_ elements only...